code/datums/mind.dm

```diff
@@ -1172,7 +1172,6 @@
 				log_admin("[key_name(usr)] has devil'ed [current]. The devil has been marked as ascendable.")
 			if("sintouched")
 				if(ishuman(current))
-					ticker.mode.sintouched += src

```

Comment:
If I am correct, |= would do that, but changing it on this line would not fix the error, I would have to change it in influenceSin() too. However, it seems to me that the convention of adding people to the antagonist ticker is by the += operator.